File: QRcodeScan2/app/src/main/java/com/example/qrcodescan2/MainActivity.java
package com.example.qrcodescan2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    Button scanBtn;
    public static TextView scanTxt;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        scanBtn = findViewById(R.id.button);
        scanTxt = findViewById(R.id.textView);
        scanBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,scaner.class));
            }
        });
    }
}

File: Kotlinkhaibodliu/app/src/main/java/com/example/kotlinkhaibodliu/MainActivity.kt
package com.example.kotlinkhaibodliu

import android.content.ContentValues.TAG
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var btn1 : Button = findViewById(R.id.button)
        btn1.setOnClickListener(
            View.OnClickListener {
                Toast.makeText(this,"btn1",Toast.LENGTH_LONG).show()
            }
        )
    }
}

